src/screens/LoginScreen/index.js

```diff
@@ -31,19 +31,20 @@ const LoginScreen = ({ navigation }) => {
 
           <View style={styles.formContainer}>
             <LoginForm onSubmit={loginRequest} />
-            <Text testID="forgot-password" style={styles.forgotPassword} onPress={handleLogin}>
-              {strings.LOG_IN.forgotPassword}
-            </Text>
-            <Text testID="connect-with-facebook" style={styles.connectFacebook} onPress={() => {}}>
-              {CONNECT_FACEBOOK_TEXT}
-            </Text>
+            <TouchableOpacity testID="forgot-password" onPress={handleLogin}>
+              <Text style={styles.forgotPassword}>{strings.LOG_IN.forgotPassword}</Text>
+            </TouchableOpacity>
+
+            <TouchableOpacity testID="connect-with-facebook" onPress={() => {}}>
```

Comment:
as @santiagorobra said, don't forget to add a //TODO comment so you don't lose track of this